### DISCUSSION TITLE
Tshock-5.0.0 Auto closes on launch, no error code.

### DISCUSSION THREAD

**Discussion by Jadebats** [Pryaxis/TShock#2818] [Q&A] [answered]
I cannot figure out why the executable won't even open the command prompt. I see a flash on the bottom toolbar as if it opens and closes immediately.

I am running windows 10, I have microsoft framework 4.8, and task manager says I still have 25.7gb of memory available. I can run the vanilla terraria server.exe fine and verified I can join that vanilla server from a separate computer. 

Since there's no error code, I'm coming up a blank. I don't know what's wrong. The server files aren't even being created (such aa the config folders). I only have the bare items from the original extracted download. The bin folder, the i18n folder, the ServerPlugins folder, and the TShock.Server.exe.

**Comment by Arthri** [comment, marked as answer]
The .NET 6 x64 Console Runtime is required https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/runtime-6.0.10-windows-x64-installer

**Reply by Jadebats** [reply]
I will check this solution. If it is true, the readme.io must be updated. Currently it sites 4.5 as the required framework.

**Reply by Jadebats** [reply]
It looks like that was the solution. After downloading the runtime, everything is running just fine. I don't know where to put the recommendation, but I would suggest the readme be updated to list this as a requirement.

Thank you for the help! It's relieved the stress I've accumulated from searching for a solution for an hour.

**Reply by RenderBr** [reply]
> I will check this solution. If it is true, the readme.io must be updated. Currently it sites 4.5 as the required framework.

the readme.io is in fact outdated, they have a new documentation page which is linked in the README: https://ikebukuro.tshock.co/
